server/validators/textValueValidator.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Simple validator function that validates a string value against a maximum length, but taking into account the fact that
+ * a new line (CRLF) in the supplied value is 2 characters.
+ * Before comparing the length, all instances of CRLF (2 characters - '\r\n') are replaced with a single '\n' character.
+ * This is non-destructive and does not change the source value itself.
+ */
+const textValueExceedsLength = (value: string, maxLength: number): boolean =>
```

Comment:
This is the new validator function that takes into account CRLF by treating it as 1 character